build.sbt

```diff
@@ -1177,7 +1182,9 @@ lazy val sbtClientProj = (project in file("client"))
       "-H:+ReportExceptionStackTraces",
       "-H:-ParseRuntimeOptions",
       s"-H:Name=${target.value / "bin" / "sbtn"}",
-    ),
+    ) ++ (if (isLinux && isArmArchitecture)
+            Seq("-H:PageSize=65536") // Make sure binary runs on kernels with page size set to 4k, 16 and 64k
+          else Nil),
```

Comment:
We need to set the page size to 64k for aarch64 binaries to make sure it runs on all kind of kernels.
I did lots of research, collected here: https://github.com/coursier/coursier/issues/2636#issuecomment-1363491391

scala-ci already merged the same fix: https://github.com/VirtusLab/scala-cli/pull/1726
coursier should follow soon: https://github.com/coursier/coursier/pull/2641